README.md

```diff
@@ -1,21 +1,18 @@
-About dask
-==========
+About dask-split
+================
 
-Home: https://dask.org/
+Home: https://github.com/dask/dask/
```

Comment:
Maybe this should point to the org?

Reply:
This is just getting picked up from the `about/home` metadata in the recipe. In particular think this is coming from `dask-core`. That said, +1 on updating all the `about/home`s to match the home URL. Thanks for flagging this

Reply:
This was fixed